protocols/Make_Callrate_FinalReport.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+#MOLGENIS walltime=05:59:00 mem=10gb ppn=6
+
+#string Project
+#list Sample_ID
+#list SentrixBarcode_A
+#list SentrixPosition_A
+#string resultDir
+#string CallrateDir
+#string gapVersion
+#string logsDir
+#string intermediateDir
+
+#Function to check if array contains value
+array_contains () {
+	local array="$1[@]"
+	local seeking=$2
+	local in=1
+	for element in "${!array-}"; do
+		if [[ "$element" == "$seeking" ]]; then
+			in=0
+			break
+		fi
+	done
+	return $in
+}
+
+module load "${gapVersion}"
+module list
+
+set -e
```

Comment:
set -e en set -u word al toegevoegd door de header, dit is nu dubbel

Reply:
waar dan?